src/commons/layouts/base/Container/Container.module.css

```diff
@@ -0,0 +1,9 @@
+.container {
+    display: inline-flex;
+    position: absolute;
+    height: 100vh;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    width: 100%;
```

Comment:
Don't you need to use 100vw here? It's only a question

Reply:
I think, in this case it does not matter what we use